src/SilverStripe/BehatExtension/Context/BasicContext.php

```diff
@@ -387,16 +411,26 @@ public function iTypeIntoTheDialog($data) {
      * @Given /^I confirm the dialog$/
      */
 	public function iConfirmTheDialog() {
-        $this->getSession()->getDriver()->getWebDriverSession()->accept_alert();
-        $this->handleAjaxTimeout();
+		try{
+			$this->getSession()->getDriver()->getWebDriverSession()->accept_alert();
+			$this->handleAjaxTimeout();
+		}catch(\WebDriver\Exception $e){
+			// have to catch and log somehow else otherwise behat result parsing fails
+			assertTrue(false, 'Dialog bug: '.$e->getMessage());
```

Comment:
Huh? Can you show how it fails? I'd rather avoid this and call `logException()` as in other places. Plus, removing the tack trace here (and having them same message in both `iConfirmTheDialog()` and `iDismissTheDialog()` will make debugging harder.